ntoskrnl/config/cmlazy.c

```diff
@@ -100,7 +100,7 @@ CmpDoFlushNextHive(_In_  BOOLEAN ForceFlush,
             /* Use another lazy flusher for this hive */
             ASSERT(CmHive->FlushCount == CmpLazyFlushCount);
             *DirtyCount += CmHive->Hive.DirtyCount;
-            DPRINT("CmHive %wZ already uptodate.\n", &CmHive->FileFullPath);
+            DPRINT("CmHive %wZ already up to date.\n", &CmHive->FileFullPath);
```

Comment:
superfluous change. Addressing this and my comment above would allow you to not touch that file at all!